src/Logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
+
+
+export default function operate(numberOne, numberTwo, operation) {
+  const operationssrt = ['+/-', '%', '÷', 'X', '-', '+'];
+  let x; let y; let
+    z;
+  switch (operation) {
+    case operationssrt[0]:
+      return new Big(numberOne).mul(-1).toFixed();
```

Comment:
I'm curious: `mul` and `times` are they the same thing, are they going to work in the way?
https://github.com/favio41/big-js#readme
I couldn't find `mul` in the doc 🤔

Reply:
well mul was the one that pull out the what i was looking on i tried lot ways. what can i say.